### ISSUE TITLE
Incompatible type for lifespan manager in v.0.1.2

### ISSUE THREAD

**Issue by Nnonexistent** [uriyyo/fastapi-lifespan-manager#121]
The last release (0.1.2) breaks the static analysis of using lifespan manager.

```python
lifespan_manager = LifespanManager()

app = FastAPI(lifespan=lifespan_manager)
```

mypy error: `Argument "lifespan" to "FastAPI" has incompatible type "LifespanManager[FastAPI]"; expected "Callable[[FastAPI], AbstractAsyncContextManager[None]] | Callable[[FastAPI], AbstractAsyncContextManager[Mapping[str, Any]]] | None"  [arg-type]`

Package versions:
* fastapi==0.109.0
* mypy==1.8.0
* fastapi-lifespan-manager==0.1.2

**Comment by uriyyo** [comment]
Thanks,
I will take a look

**Comment by uriyyo** [comment]
Hi @Nnonexistent,

This issue should be fixed in version `0.1.3`. Could you please confirm that this issue is fixed on your side?

**Comment by Nnonexistent** [comment]
Hi @uriyyo, 

yes, I confirm, that the issue is fixed in v0.1.3.

Thank you for the fast response!

**Comment by uriyyo** [comment]
Great, thanks for the fast feedback. Have a nice day!
